oddbird/oddleventy
# pwa-article

![](https://images.unsplash.com/photo-1534628337197-3f1738d13421?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwxfDB8MXxyYW5kb218MHx8Y3V0ZSxhbmltYWwsQ0hBTkdFX01FfHx8fHx8MTY5ODI5MzYyMg&ixlib=rb-4.0.3&q=80&)




## Steps to test/reproduce
Review all three PWA comparison articles in the series to make sure the linking is working properly.
Review PWA article for typos and accurate content.

https://deploy-preview-437--oddleventy.netlify.app/2023/10/26/when-to-choose-a-progressive-web-app/
